vault/logical_system.go

```diff
@@ -4601,8 +4596,6 @@ func (b *SystemBackend) pathInternalOpenAPI(ctx context.Context, req *logical.Re
 		return nil, err
 	}
 
-	doc.CreateOperationIDs(context)
```

Comment:
If you remove this, existing compiled plugins, using older versions of the `sdk/framework` package, will now have no `operationId`s at all.

Reply:
You are correct, and good catch I did not realize that this was outside of `sdk/framework`. I will add back `CreateOperationIDs` and mark it as deprecated.